core/src/main/scala/kanaloa/reactive/dispatcher/queue/QueueProcessor.scala

```diff
@@ -21,6 +21,11 @@ class QueueProcessor(
   resultChecker:          ResultChecker
 ) extends Actor with ActorLogging with MessageScheduler {
 
+  val healthCheckSchedule = {
```

Comment:
So, I'm not entirely sure what this is supposed to change?  We were already listening for Worker failures, and checked to see if this dipped below the minimum.  Since this check is already event driven(by a Worker's death), I don't see how this really make any improvement.  If anything, doesn't this allow for windows where there are no Workers alive, and none being created?

To fix the case where on startup, Routees are failing to be created(and ultimately failing to create any Workers), couldn't we just add a check in the `RouteeFailed` case, to see if we have 0 `currentWorkers`, and take action similar to `Terminated(worker)`?